src/main/java/nl/novi/techiteasy/controllers/TelevisionController.java

```diff
@@ -22,7 +23,7 @@ public ResponseEntity<String> addTelevision(@RequestBody Television television)
     }
 
     @PutMapping("/televisions/{id}")
-    public ResponseEntity<String> updateTelevision(@PathVariable String id @RequestBody Television television) {
+    public ResponseEntity<String> updateTelevision(@PathVariable String id, @RequestBody Television television) {
         return ResponseEntity.noContent().build();
```

Comment:
Je had in deze opdracht al heel goed de repository aan je controller kunnen koppelen. Zo had je bij een PostMapping bijvoobeeld al werkelijk een Television in de database kunnen saven en dan had je je eigen database live in actie kunnen zien. Nu doet je controller nog niet zo veel. De structuur die je hier opzet is echter al wel goed.